### ISSUE TITLE
Control labels not reported to accessibility technologies (or missing altogether)

### ISSUE THREAD

**Issue by tyrylu** [Reverp/Toxy#24]
Hello.
When attempting to use Toxy with a screen reader (NVDA, etc.), the screen reader can not find the labels for the various buttons (status etc.). The edit boxes also have no label reported for them. Could some dev look into this? It would be really nice to have some accessible Tox client.
